docs/_source/docs/resolvers.rst

```diff
@@ -187,7 +187,7 @@ setup.py
     setup(
         name='custom_resolver',
         entry_points={
-            'sceptre.hooks': [
+            'sceptre.resolvers': [
```

Comment:
When you rebase you will find this conflicts with the latest docs. It might be easier when you rebase to drop this change.